announcements/_posts/2019-03-29-w10-programming-languages.md

```diff
@@ -0,0 +1,33 @@
+---
```

Comment:
In the past we've included the date we post in the filename.

Reply:
You also have to include the date in the announcement emails header because it is treated as an event. We still do the date in the file name.

Reply:
Right, I guess I'm saying I think the "post" date should be today, ie what goes in the filename. Of course the event date is tomorrow.